src/containers/Tenant/Diagnostics/Healthcheck/Healthcheck.tsx

```diff
@@ -61,15 +73,15 @@ export const Healthcheck = (props: HealthcheckProps) => {
         if (data && data['self_check_result']) {
             return preview ? (
                 <Preview
-                    data={data}
+                    issuesTrees={issuesTrees}
```

Comment:
Is there a point in passing the whole trees there? Only the roots are displayed.
There also may be no point in creating a selector to only select the roots. This is a question to consider and prove, not to blindly fix.

Reply:
I think it makes sense since trees can be big. Also in this case we work with the whole trees only in reducer while components get either roots or single tree